src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -65,11 +72,11 @@ export const NewProposal = ({ navigation, data, query }) => {
     setValue
   } = useForm({
     defaultValues: {
-      title: data?.title,
-      description: data?.description,
-      tagList: data?.tagList?.toString(),
-      summary: data?.summary,
-      videoUrl: data?.videoUrl
+      title: data?.title ? data?.title : null,
+      description: data?.description ? data?.description : null,
+      tagList: data?.tagList ? data?.tagList?.toString() : null,
+      summary: data?.summary ? data?.summary : null,
+      videoUrl: data?.videoUrl ? data?.videoUrl : null
```

Comment:
again, why have you extended the conditions here to return `null`s instead of `undefined`s?